scss/_variables.scss

```diff
@@ -321,6 +321,33 @@ $theme-colors: (
 ) !default;
 // scss-docs-end theme-colors-map
 
+$primary-text:            $blue-600 !default;
```

Comment:
For all `$*-text`, `$*-bg-subtle` and `$*-border-subtle`, IMO they shoud be presented in the documentation (maybe included in `// scss-docs-start theme-color-variables`).

If we're thinking of projects that customize the colors, they will probably need to remap those vars with other vars.

For example, changing `$primary` to set `$yellow` won't update all the colors in Bootstrap anymore. The user will need to choose a value for `$primary-text`, `$primary-bg-subtle` and `$primary-border-subtle` as well.

And also the `$primary-*-dark` if the project needs dark mode.